lockfile/src/parsers/mod.rs

```diff
@@ -1,12 +1,11 @@
 use nom::branch::alt;
 use nom::bytes::complete::{tag, take, take_until};
-use nom::character::complete::{line_ending, multispace0, none_of, space0};
+use nom::character::complete::{line_ending, multispace0, space0};
 use nom::combinator::{eof, opt, recognize};
-use nom::error::{context, ParseError, VerboseError};
+use nom::error::{context, VerboseError};
 use nom::multi::{count, many1, many_till};
 use nom::sequence::{delimited, tuple};
 use nom::{AsChar, IResult};
-use phylum_types::types::package::{PackageDescriptor, PackageType};
```

Comment:
Does the lockfile crate still use `phylum_types` somewhere else or can we remove it as a dependency now?